### DISCUSSION TITLE
intipaddr="10.0.0"

### DISCUSSION THREAD

**Discussion by TwiztedWisard** [Zackptg5/Wireguard-Pi-Hole-Cloudflared-Unbound-DNSCrypt-VPN-Server#5] [Q&A] [answered]
I am in no way a linux expert but I wanted to confirm the variable intipaddr. You have it set at 10.0.0.....should this be 10.0.0.0? if not, why? I'm all about the learning ;)

**Comment by Zackptg5** [comment, marked as answer]
You're correct that the full ip would be 10.0.0.0 but I set the variable the way I did intentionally. The reason why is that there's several places in the script where the intipaddr variable is reused for different things such as wireguard clients and pihole ip. Wireguard client ips would be 10.0.0.2, 10.0.0.3, etc. Because I set the variable to 10.0.0, I can reuse it here while if I had it set to the full/true internal ip address, I wouldn't be able to without modifying the variable or creating another one

So for calling the variable for normal internal ip address, I use $intipaddr.0
For pihole ip: $intipaddr.1
For wireguard client: $intipadd.2
etc
:)

**Reply by TwiztedWisard** [reply]
Thanks for the response man! And a speedy one at that! :D As I said I'm all about the learning and your explanation was spot on...this did highlight one flaw tho....oh, not in your code...no no....I will now be writing the line "MUST READ CODE THOROUGHLY" a thousand times! :D

**Reply by Zackptg5** [reply]
Haha, no problem!
